core/util/BUILD

```diff
@@ -96,6 +112,7 @@ pkg_tar(
         "//core/util:Exception.h",
         "//core/util:prelude.h",
         "//core/util:jit_util.h",
-        "//core/util:trt_util.h"
+        "//core/util:trt_util.h",
+        "//core/util:converter_util.h"
```

Comment:
I guess once we replace the above uses of arrToTensor with tensor_to_const, we don't need the `converter_util.h` header file right ?

Reply:
I also relpace arrToTensor with tensor_to_const and remove the converter_util.h